src/varity/ref_gene.clj

```diff
@@ -73,19 +73,24 @@
 
 (def ^:private pos-index-block 1000000)
 
+(def max-promoter-size 10000)
```

Comment:
In genomics, "promoter" means a region of DNA which located near upstream from the transcription start sites of genes.

Thus`max-promoter-size` does not make sense. I think `margin` or `transcription-margin` is correct words in here.